src/makeCalculator.js

```diff
@@ -4,7 +4,44 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function (value) {
+      this.result += value;
+
+      return this;
+    },
```

Comment:
Instead of using the `function` keyword, you can use the method shorthand syntax to define methods within the `calculator` object.